components/checkbox/stories/checkbox.stories.js

```diff
@@ -130,7 +130,152 @@ const CheckboxGroup = (args) => html`
 	</div>
 `;
 
+// AllVariantsCheckboxGroup displays the default, invalid, disabled and read-only variations as a single
+// story for Standard and Emphasized variants
+const AllVariantsCheckboxGroup = (args) =>  html`
+	<div style="display: flex;">
+    <div style="display: flex; flex-direction: column; padding: 16px">
+    	<h4>Default</h4>
+      ${Template({
+        ...args,
+        iconName: undefined,
+      })}
+      ${Template({
+        ...args,
+        isChecked: true,
+      })}
+      ${Template({
+        ...args,
+        isIndeterminate: true,
+      })}
+        ${Template({
+        ...args,
+        isDisabled: true,
+      })}
+      ${Template({
+        ...args,
+        label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+        customStyles: { "max-inline-size": "200px" },
+      })}
```

Comment:
You could simplify the repetition of these 5 Template functions below by using another template function within this one; I think you could probably reuse the existing `CheckboxGroup`? If you want to pair on that, let me know.

Reply:
oooooo ok- I figured there was probably a way to simplify it! I might take you up on a pairing session. I'll see if I can figure that out today and slack you if I can't figure it out.

Reply:
How's this looking? More like what you had in mind? https://github.com/adobe/spectrum-css/pull/2793/commits/b19b50b50d4a182808f7f7b0c87b4e1e2a20e172